serpentTools/parsers/depletion.py

```diff
@@ -130,3 +133,27 @@ def _processChunk(self, chunk, name, variable):
                     continue
                 cleaned.append(line[:line.index('%')])
         self.materials[name].addData(variable, cleaned)
+
+    def _precheck(self):
+        """do a quick scan to ensure this looks like a material file
+        """
+        with open(self.filePath) as fh:
+            for line in fh:
+                sline = line.split()
+                if sline == []:
+                    continue
+                elif 'MAT' == line.split()[0]:
+                    self.materialCount += 1
+        if self.materialCount == 0:
```

Comment:
See above comment about not making this an instance attribute

Reply:
done!